### PR TITLE
Added small table with quicklinks to docs

### PR DESCRIPTION
**Pull request by danieldobalian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
-﻿#Microsoft Azure Active Directory Authentication Library (ADAL) for Android
+﻿# Microsoft Azure Active Directory Authentication Library (ADAL) for Android
 ===========
 
+| [Code Samples](https://github.com/azure-samples?utf8=✓&q=active-directory-android) | [Reference Docs](http://javadoc.io/doc/com.microsoft.aad/adal/1.11.0) | [Developer Guide](https://aka.ms/aaddev)
```
Comment (weijjia) [inline README.md]: @danieldobalian Are you manually generating the javadoc per release?

Comment (danieldobalian) [inline README.md]: We considered it, but found this site already solved the problem.  When we do a release, this site generates the docs and hosts it.  For now we'll have to update this link. The site supports "latest" via `.../adal/`, but currently directs to 2.0.4 alpha.

Comment (weijjia) [inline README.md]: adal will stay on 1.x I believe, can we point it to always pull in the latest 1.x version?

Comment (danieldobalian) [inline README.md]: I'll look into the publishing system they use and get back to you.

Comment (danieldobalian) [inline README.md]: @weijjia can we merge and I'll handle publishing problems offline?

Comment (danieldobalian) [inline README.md]: @addev-ashish

Comment (addev-ashish) [inline README.md]: can you make this 1.12.0

**Comment by danieldobalian** [comment]
@addev-ashish @nazukj Closing this and will re-open once we scrub javadocs.

**Pull request closed without merging**